client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
```

Comment:
When #16776 lands, the button will stop shrink, and the image won't float for small screens.